docs/content/topics/administrators/_index.md

```diff
@@ -3,9 +3,26 @@ title: Administrators
 description: Brigade system administrators
 aliases:
   - /index.md
+  - /topics/administrators
   - /topics/administrators/index.md
 ---
 
 # Brigade Administrators
 
-TODO
\ No newline at end of file
+Users who are in charge of authentication and authorization concerns within the
+Brigade system are known as Administrators.
```

Comment:
I'm worried this definition isn't general enough.

What if we present admins as users who are responsible for _managing_ an instance of Brigade?

Maybe also worth noting that this role may often, but not always, be fulfilled by the same person/people fulfilling the operator role?

Reply:
The concern about this not being a general enough description repeats in a few other spots.

Reply:
Indeed, I added the following text to the roles doc:

```
Naturally, there may be overlap between these roles, and for development
setups, one or two users might cover all of them.  However, they also serve
nicely as categories of context for documentation, so without further ado,
let's dive in.
```

But I can certainly add role-specific notes to each index, such as the one you've pointed out here, where the admin role might naturally be filled by the operator.